zcash_client_backend/Cargo.toml

```diff
@@ -20,12 +20,17 @@ base64 = "0.13"
 ff = "0.11"
 group = "0.11"
 hex = "0.4"
+hdwallet = { version = "0.3.0", optional = true }
 jubjub = "0.8"
+log = "0.4"
 nom = "7"
 percent-encoding = "2.1.0"
 proptest = { version = "1.0.0", optional = true }
 protobuf = "2.20"
 rand_core = "0.6"
+ripemd160 = { version = "0.9.1", optional = true }
+secp256k1 = { version = "0.20", optional = true }
```

Comment:
The most recent version is 0.21.2 ([crates.io](https://crates.io/crates/secp256k1/versions), [CHANGELOG](https://github.com/rust-bitcoin/rust-secp256k1/blob/master/CHANGELOG.md)).

Reply:
secp256k1 will have to be upgraded in concert across multiple dependencies before v4.7.0, so I'm not doing that here.